jsinterop-ts-defs-processor/src/main/java/com/vertispan/tsdefs/visitors/ClassMethodVisitor.java

```diff
@@ -18,14 +18,22 @@
 import com.vertispan.tsdefs.HasProcessorEnv;
 import com.vertispan.tsdefs.builders.HasFunctions;
 import com.vertispan.tsdefs.builders.TsElement;
+import com.vertispan.tsdefs.model.TsDoc;
 import com.vertispan.tsdefs.model.TsMethod;
 import javax.lang.model.element.Element;
 import javax.lang.model.element.ExecutableElement;
 
 public class ClassMethodVisitor<T> extends TsElement {
+  private final boolean copyDocs;
 
   public ClassMethodVisitor(Element element, HasProcessorEnv env) {
     super(element, env);
+    this.copyDocs = true;
```

Comment:
what's the use case where we want it to still copy docs?

Reply:
This can be called for the top level types and the child types, so we are using the same class for both, the other change set this to false only if we are visiting inherited methods and not methods that are actually in defined in the current type.